tcms/rpc/tests/test_testrun.py

```diff
@@ -312,3 +427,52 @@ def test_wrong_date_format(self):
         self.assertNotEqual(update_fields['summary'], test_run.summary)
         self.assertNotEqual(datetime.strptime(update_fields['stop_date'], '%d-%m-%Y'),
                             test_run.stop_date)
+
+    def test_update_with_product(self):
+        test_run = TestRunFactory()
+        product = ProductFactory()
+        updated_test_plan = TestPlanFactory(product=product)
+        updated_build = BuildFactory(product=product)
+        updated_summary = 'Updated summary.'
+        updated_stop_date = datetime.strptime('2020-05-05', '%Y-%m-%d')
+
+        update_fields = {
+            'plan': updated_test_plan.pk,
+            'product': product.id,
+            'build': updated_build.pk,
+            'summary': updated_summary,
+            'stop_date': updated_stop_date
+        }
+        updated_test_run = self.rpc_client.TestRun.update(test_run.pk, update_fields)
```

Comment:
You can also move the `update_fields` dictionary as the second argument here and remove the intermediate variable b/c you don't use it anywhere else.